frontend-react/src/components/Content/MarkdownDirectory.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { ElementDirectory, MarkdownDirectory } from "./MarkdownDirectory";
+
+const TestDiv = <div>Hello, world</div>;
+const TestElement = () => TestDiv;
+
+describe("ContentDirectory extnesions", () => {
```

Comment:
[sp] `extnesions`

Reply:
I swear I learned how to spell as a child 😂